### DISCUSSION TITLE
Forward images via FTP

### DISCUSSION THREAD

**Discussion by MALAONE1** [jekhokie/raspberry-noaa-v2#351] [Ideas]
Thank you for your great software.  Would it be possible to set up an FTP forwarding of the images?  Thank you very much for your work

**Comment by noaahunter** [comment]
I have installed Rust on my Pi and coded an little ftp uploader and its work great.
curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh

**Reply by LloydOS** [reply]
Hi, do you have any instructions on how to set this up?

**Reply by ve3elb** [reply]
@noaahunter can you share your FTP files and install instructions I would like to add it to my setup. Thank you. 

There is one built into WXtoimg am not sure why it was not implemented in Raspberry NOAA V2￼

**Reply by dom-robinson** [reply]
The wxtoimg commandline has a limited (sub)set of instructions to the GUI driven one.

**Comment by airlomba** [comment]
Hello all,
That's a nice feature that I also would like a lot to see it implemented in the future.
Thank you for your work.
Best regards.
